GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -296,6 +296,7 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(FormBrowse.Command.QuickPush, Keys.Control | Keys.Shift | Keys.Up),
                     Hk(FormBrowse.Command.Rebase, Keys.Control | Keys.Shift | Keys.E),
                     Hk(FormBrowse.Command.Stash, Keys.Control | Keys.Alt | Keys.Up),
+                    Hk(FormBrowse.Command.StashStaged, Keys.None),
```

Comment:
Should be moved down one line in order have it sorted alphabetically.

Reply:
❔ 
```suggestion
                    Hk(FormBrowse.Command.StashStaged, Keys.Control | Keys.Shift | Keys.Alt | Keys.Up),
```

Reply:
done.
I didn't want to give a default shortcut that complex without testing it because I don't know why I don't have the default GitExtensions shortcuts (and I prefer them!) but it's OK for me....